server/methods/notifications/pushbulletNotification.js

```diff
@@ -8,37 +8,60 @@ Meteor.methods({
 
         var access_token = settings.pushbulletAPI
         var channel_tag = settings.pushbulletChannel
+        var device_id = settings.pushbulletDeviceID
         var pushbullet_url = 'https://api.pushbullet.com/v2/pushes'
         var payload = {}
 
         //////// Create JSON object of http.post Parameters
         if(typeof channel_tag === 'undefined' ) {
```

Comment:
Ok, so now that we are looking at having the possibility of an either/or scenario that would simply add an additional key/value to the params, it is probably better to go ahead and define the params with the required keys filled out and then simply add the device id or channel to the base parameters. That way, we clean up the code a bit by not defining the variable repeatedly.

Ex. (Don't mind the pseudo code)
```
var payload = {
                headers: {
                    'Access-Token': access_token
                },
                params: {
                    type: 'note',
                    title: title,
                    body: body
                },
        }

if channel_tag && device_iden{
    console.log("Only one or the other, not both")
    return false
}
if channel {
    payload['params']['channel_tag'] = channel_tag
} else {
    payload['params']['device_iden'] = device_iden
}
```
This cuts down the lines of code considerably by not defining the payload 3 times and makes it much easier to read.